src/services/backend.js

```diff
@@ -61,22 +49,20 @@ const createUploadSession = (
         betaIp: API.BROKER_NODE_B,
         storageLengthInYears
       })
-      .then(({ data }) => {
-        console.log("UPLOAD SESSION SUCCESS: ", data);
-        const { id: alphaSessionId, betaSessionId } = data;
-        const { invoice } = data;
-        resolve({ alphaSessionId, betaSessionId, invoice });
+      .then(({data}) => {
+        const {id: alphaSessionId, betaSessionId} = data;
```

Comment:
No need to change this, but we could probably do:
`const {id: alphaSessionId, betaSessionId, invoice} = data;`